x-pack/test/functional/apps/infra/constants.ts

```diff
@@ -24,6 +24,8 @@ export const DATES = {
       withoutData: '10/09/2018 10:00:00 PM',
       min: '2018-10-17T19:42:21.208Z',
       max: '2018-10-17T19:58:03.952Z',
+      processesDataStartDate: '2023-03-28T18:20:00.000Z',
+      processesDataEndDate: '2023-03-28T18:21:00.000Z',
```

Comment:
**question:** why don't we use a single date range for all the dataset? Having multiple time range can complicate our tests and fake data inconsistencies. We could keep the min/max timerange and adapt the esArchiver data to be in that range.

Reply:
This is a bit of a special case I think - I want to see **only** the data coming from this data set so I used different dates - the background is that if we have other hosts from another dataset they are incomplete and don't include the `process.pid` so once the process tab is opened it will return an error (maybe some sort of filtering can be applied to avoid that I was just thinking that having only the host with the correct data for this test returned will prevent flaky results ) - so this way I know that only the host from this dataset will be returned. 
Another option was to edit all the existing data/mappings in the existing dataset but that will be a lot of manual editing and risk breaking existing tests.